sml_small/utils/error_utils.py

```diff
@@ -77,3 +77,22 @@ def get_param_outside_range_error(tag: str, param_list: List[str]) -> str:
     param_list_text = separator.join(param_list)
 
     return f"{tag} is outside of range {param_list_text}"
+
+
+def get_boundary_error(param_list: List[float]) -> str:
```

Comment:
This may need to change as Ellie has asked for lower threshold greater or equal to upper threshold to be treated as error

Reply:
Amended, Error now reads Lower limit is larger than or equal to the upper limit {lower:upper}